test/Staking.test.js

```diff
@@ -5,21 +5,23 @@ const {
   balance,
   constants
 } = require('@openzeppelin/test-helpers')
+const upgrades = require('@openzeppelin/truffle-upgrades')
 const { toBN, asciiToHex, padRight } = require('web3-utils')
 const expect = require('chai').expect
 
 const Staking = artifacts.require('Staking')
 const StorageManager = artifacts.require('StorageManager')
 const ERC20 = artifacts.require('MockERC20')
 
-contract('Staking', ([staker, randomPerson]) => {
+contract('Staking', ([randomPerson, staker]) => {
```

Comment:
why do you revert this?

Reply:
The plugins have still an open issue that is that you cannot specify the `from` address. It uses the `accounts[0]` from the accounts that Truffle provides. So I needed the first account to be the one Deploying the proxies. The easiest way was to revert the order to achieve that and have the one that deploys first.

Reply:
Ahh. Ok, make sense now). Thanks

Reply:
https://github.com/OpenZeppelin/openzeppelin-upgrades/issues/85